schema/ingestion_config/v1.0.0/ingestion_config_models_extended.py

```diff
@@ -900,6 +902,19 @@ def valid_sources(cls: Self, source_list: List[GainSource]) -> List[GainSource]:
         return validate_sources(source_list)
 
 
+# ==============================================================================
+# Identified Object Validation
+# ==============================================================================
+class ExtendedValidationIdentifiedObject(IdentifiedObject):
+    @model_validator(mode="after")
+    def validate_identified_object(self) -> Self:
+        if re.match(GO_ID_REGEX, self.object_id):
+            validate_id_name_object(self, self.object_id, self.object_name, id_field_name="object_id", validate_name=True, ancestor=CELLULAR_COMPONENT_GO_ID)
+        elif re.match(UNIPROT_ID_REGEX, self.object_id):
+            validate_id_name_object(self, self.object_id, self.object_name, id_field_name="object_id", validate_id_function=validate_uniprot_id)
+        return self
+
```

Comment:
As all this data will be a part of the csv file, I am confused on why we are validating that, as a part of the ingestion config. 

I think we are missing the update to the ingestion config, to allow for the new block `identified_objects`